build-system/test-configs/config.js

```diff
@@ -162,7 +162,6 @@ const prettifyGlobs = [
   '.github/workflows/continuous-integration-workflow.yml',
   '**/*.json',
   '**/OWNERS',
-  '**/*.md',
```

Comment:
Would this change exclude all markdown files from Prettify?

Reply:
Have you looked at whether Prettier config can be tweaked for markdown instead, such as with the [`tabWidth`](https://prettier.io/docs/en/options.html#tab-width) config setting?

Reply:
ah! looks like prettier can use [eslint style overrides](https://prettier.io/docs/en/configuration.html#configuration-overrides) - thanks for the push on thisQ

Reply:
Yep. Our markdown settings live here:

https://github.com/ampproject/amphtml/blob/59d57b3c993f3de39d17f2cae0c5802efcf7cd81/.prettierrc#L11-L14

Reply:
thanks - pushed an update :D